fix_doc_links.py

```
@@ -0,0 +1,13 @@
+import os
+import glob
+import re
+
+for filename in glob.glob('book/src/content/docs/**/*.md', recursive=True) + glob.glob('book/src/content/docs/**/*.mdx', recursive=True):
+    with open(filename, 'r') as f:
```

Ruff (0.15.10)

[warning] Unnecessary mode argument

Remove mode argument

(UP015)